config/routes.rb

```diff
@@ -2,6 +2,9 @@
   root to: "users#index"
 
   resources :users, only: [:index, :show] do 
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :new, :create] do 
+      resources :comments, only: [:create]
+      resources :likes, only: [:create]
+    end
```

Comment:
- [ ] After making the requested changes listed above 🙃 Don't forget to update with a pair of routes in the config/routes.rb file to create comments.
- eg:

```ruby
   POST   /posts/:post_id/comments(.:format)     comments#create
   GET   /posts/:post_id/comments/new(.:format)     comments#new
```

Reply:
Hey @michael-duke, you only need a create action router

Reply:
Thanks for confirming @BenMukebo , I requested for review :smile: